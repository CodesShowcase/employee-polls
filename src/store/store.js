import { configureStore } from '@reduxjs/toolkit';
import baseReducer from '../reducers';
import logger from 'redux-logger';
import thunk from "redux-thunk";
import monitorReducerEnhancer from '../enhancers/monitorReducers';

const store = configureStore({ reducer: baseReducer, middleware: (getDefaultMiddleware) => getDefaultMiddleware().concat(logger).concat(thunk), enhancer: monitorReducerEnhancer });

export default store;
