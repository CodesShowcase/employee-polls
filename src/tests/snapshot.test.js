import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import { screen } from '@testing-library/react';
import store from '../store/store';
import Navigation from '../components/Nav/Nav';

describe('Snapshot | 1. Navbar Snapshot', () => {
  test('Snapshot | Navbar should render', () => {
    const component = renderer
      .create(
        <MemoryRouter>
          <Provider store={store}>
            <Navigation />
          </Provider>
        </MemoryRouter>
      )
      .toJSON();

    expect(component).toMatchSnapshot();
  });
});
