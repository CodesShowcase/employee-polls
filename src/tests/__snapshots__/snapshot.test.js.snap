// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot | 1. Navbar Snapshot Snapshot | Navbar should render 1`] = `null`;
